config/database.yml

```diff
@@ -4,7 +4,10 @@
     if database_config.rdsCa
       rdsCaFile = File.new('tmp/rdsCa', 'wt')
       rdsCaFile.write(database_config.rdsCa)
-      database_config.rdsCaPath = File.expand_path(rdsCaFile.path)
+      database_config.sslrootcert = File.expand_path(rdsCaFile.path)
+      database_config.sslmode = 'verify-full'
+    else
+      database_config.sslmode = 'disable'
```

Comment:
Please default to `prefer`. Even `require` could work on standard envs (dev, CI).

Reply:
well I disabled it because on Clowder deployments on ephemeral environments it won't provide SSL at all. And `prefer` is not recommended anyways. 

> The default value for sslmode is prefer. As is shown in the table, this makes no sense from a security point of view, and it only promises performance overhead if possible. It is only provided as the default for backward compatibility, and is not recommended in secure deployments.

But ... yeah- maybe `prefer` is better than `disable` :)